netty/src/main/java/io/grpc/netty/NettyClientHandler.java

```diff
@@ -484,7 +484,7 @@ private void createStream(CreateStreamCommand command, final ChannelPromise prom
     final int streamId;
     try {
       streamId = incrementAndGetNextStreamId();
-    } catch (StatusException e) {
+    } catch (StatusRuntimeException e) {
```

Comment:
It's unfortunate that Exceptions are being used a flow control for a normal event.